guides/w3c_validator.rb

```diff
@@ -32,23 +32,19 @@
 module RailsGuides
   class Validator
     def validate
-      validator = MarkupValidator.new
+      # https://github.com/w3c-validators/w3c_validators/issues/25
+      validator = NuValidator.new(validator_uri: "https://validator.w3.org/nu/")
```

Comment:
It looks this it is [already pointing](https://github.com/w3c-validators/w3c_validators/blob/ac232091b7691721b96890c1bdd61d58c2b51ec9/lib/w3c_validators/nu_validator.rb#L5) to this URL so there may be no need to specify the URL.